malariagen_data/anoph/snp_data.py

```diff
@@ -1475,10 +1514,13 @@ def biallelic_snp_calls(
     @check_types
     @doc(
         summary="Load biallelic SNP genotypes.",
-        returns="""
-            An array of shape (variants, samples) where each value counts the
-            number of alternate alleles per genotype call.
-        """,
+        returns=dict(
+            gn="""
+                An array of shape (variants, samples) where each value counts the
+                number of alternate alleles per genotype call.
+            """,
+            samples="Sample identifiers.",
+        ),
```

Comment:
Need to return samples as well to allow for alignment with metadata later, especially if any downsampling has occurred.